lib/core/src/lib/auth/services/authentication.service.spec.ts

```diff
@@ -23,6 +23,11 @@ import { setupTestBed } from '../../testing/setup-test-bed';
 import { CoreTestingModule } from '../../testing/core.testing.module';
 import { BasicAlfrescoAuthService } from '../basic-auth/basic-alfresco-auth.service';
 import { OidcAuthenticationService } from '../oidc/oidc-authentication.service';
+// import { RedirectAuthService } from '../oidc/redirect-auth.service';
```

Comment:
do we need this line?

Reply:
Nope 😬 Sorry, my bad, I should have removed it!
Thank man